demos/clarad.js

```diff
@@ -0,0 +1,30 @@
+const axios = require('axios');
+
+// Print out a list of names in Gwen's repo. Can use other endpoints too, such as, id.
+// Make a call to the Github API chaining .then() and .catch() onto the Promises and print out your results.
+axios.get('https://api.github.com/users/gwenf/repos')
+    .then(response => {
+        response.data.forEach(element => console.log(element.name));
+    })
+    .catch(error => {
+        console.log(error);
+    });
+
+// Now create an async function and make the call using await syntax
+async function axiosAsync () {
+    // await response of fetch call
+    let response = await axios.get('https://api.github.com/users/gwenf/repos');
+    // only proceed once promise is resolved
+    let data = await response;
```

Comment:
There is no second promise here. The axios call is a promise, but `response` is just a variable after it gets populated with the data sent back from the Axios call. You can remove this line.